tools/sl4a_client.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3.4
+#
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tool to interactively call sl4a methods.
+
+Usage:
+$ sl4a_client
+Connected to 008d1d9749813804. Call sl4a methods against 'd' or 'droid'
+>>> droid.telephonyGetSimOperatorName()
```

Comment:
How about using a less special call in the example, say "getBuildID".
Not all devices have telephony support, but all devices probably have build id